tests/functional/splinter_test.c

```diff
@@ -223,11 +223,9 @@ test_trunk_lookup_thread(void *arg)
       for (uint8 spl_idx = 0; spl_idx < num_tables; spl_idx++) {
          if (test_is_done(done, spl_idx))
             continue;
-         platform_throttled_error_log(
-            DEFAULT_THROTTLE_INTERVAL_SEC,
-            PLATFORM_CR "lookups %3lu%% complete for table %u",
-            lookup_base[spl_idx] / (total_ops[spl_idx] / 100),
-            spl_idx);
+         platform_error_log(PLATFORM_CR "lookups %3lu%% complete for table %u",
```

Comment:
Again, why was throttled removed? (Ref issue #336, which suggests ways to fix this throttling mechanism)